crates/turborepo-lib/Cargo.toml

```diff
@@ -26,23 +26,38 @@ axum-server = "0.4.4"
 chrono = { version = "0.4.23", features = ["serde"] }
 clap = { version = "4.0.22", features = ["derive"] }
 clap_complete = "4.0.6"
+command-group = { version = "2.0.1", features = ["with-tokio"] }
```

Comment:
From the same effort:
https://github.com/watchexec/watchexec

In case we can learn from them.